errata_tool/bug.py

```diff
@@ -0,0 +1,40 @@
+from errata_tool import ErrataConnector
+
+
+class Bug(ErrataConnector):
+    def __init__(self, id):
+        """
```

Comment:
docstring linting prefers no leading whitespace between """ and first word of summary